tests/phpunit/Patcher/ListPatcherTest.php

```diff
@@ -91,9 +92,28 @@ public function patchProvider() {
 
 		$argLists[] = array( $patcher, $base, $diff, $expected );
 
+		$patcher = new ListPatcher();
+		$base = array(
+			$this->newObject( 'foo' ),
+			$this->newObject( 'bar' ),
+		);
+		$diff = new Diff( array(
+			new DiffOpRemove( $this->newObject( 'foo') ),
```

Comment:
Spaceeeeeeeeeeeeee

Reply:
fixed on master skipping the PR. PHPCS wake up!